robotd/devices.py

```diff
@@ -243,8 +241,14 @@ class ServoAssembly(Board):
     INPUT = 'hi-z'
 
     @classmethod
+    def included(cls, node):
+        return node['ID_MODEL_ID'] in ['0043', '7523'] and node['ID_VENDOR_ID'] in ['2a03', '1a86']
+
+    @classmethod
     def name(cls, node):
         """Board name."""
+        if node['ID_VENDOR_ID'] == '2a03':  # real arduino
+            return str(node['ID_SERIAL_SHORT'])
         return 'SB_{}'.format(node['MINOR'])
```

Comment:
I would prefer pushing to use `ID_SERIAL_SHORT` where it's available perhaps—maybe `.get` here?